src/components/BooksList/BooksList.jsx

```diff
@@ -23,7 +24,7 @@ const BooksList = (props) => {
         </thead>
         <tbody>
           {
-            books.map(book => <Book key={uuid()} id={book.id} title={book.title} category={book.category} />)
+            books.map(book => <Book key={uuid()} {...book} click={handleRemoveBook} />)
```

Comment:
if this method was called something else other than `click`, i think it will make more sense, especially when you are using it in the `<BOOK />` component

Reply:
Thanks for the observation!